components/calendar/index.css

```diff
@@ -86,7 +89,7 @@ governing permissions and limitations under the License.
 
 .spectrum-Calendar-prevMonth,
 .spectrum-Calendar-nextMonth {
-  transform: logical rotate(0deg); /* stylelint-disable-line declaration-property-value-no-unknown */
+  transform: var(--spectrum-logical-rotation,);
```

Comment:
Should this be either `transform: var(--spectrum-logical-rotation,) rotate(0deg);` or `transform: var(--spectrum-logical-rotation);` without the comma? With the comma and nothing after the var() it seems like it would be creating `transform:;`.

Reply:
I just pushed up a change without the comma, let’s see if that works